Resources/views/Gallery/index.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-{% sonata_template_box 'This is the gallery index template. Feel free to override it.' %}
+{% sonata_template_box 'This is the gallery index template. Feel free to override it.' 'SonataMediaBundle'%}
```

Comment:
This will always call the translator, also we don't have any translation

Reply:
I would say yes 👍